Remotion/Web/UnitTests/Core/UI/Controls/CommandTests/EventCommandTest.cs

```diff
@@ -87,16 +90,60 @@ public void Render_WithAccessGranted ()
       Assert.That(_testHelper.HtmlWriter.Tag, Is.Not.Null, "Missing Tag");
       Assert.That(_testHelper.HtmlWriter.Tag, Is.EqualTo(HtmlTextWriterTag.A), "Wrong Tag");
 
-      Assert.That(_testHelper.HtmlWriter.Attributes[HtmlTextWriterAttribute.Href], Is.Not.Null, "Missing Href");
-      Assert.That(_testHelper.HtmlWriter.Attributes[HtmlTextWriterAttribute.Href], Is.EqualTo("fakeFallbackUrl"), "Wrong Href");
+      Assert.That(
+          _testHelper.HtmlWriter.Attributes.Keys,
+          Is.EquivalentTo((object[])[
+              HtmlTextWriterAttribute.Href, HtmlTextWriterAttribute.Onclick, HtmlTextWriterAttribute.Title, HtmlTextWriterAttribute2.Role, HtmlTextWriterAttribute2.OnKeyUp]));
 
-      Assert.That(_testHelper.HtmlWriter.Attributes[HtmlTextWriterAttribute.Onclick], Is.Not.Null, "Missing OnClick");
+      Assert.That(_testHelper.HtmlWriter.Attributes[HtmlTextWriterAttribute.Href], Is.EqualTo("fakeFallbackUrl"), "Wrong Href");
       Assert.That(_testHelper.HtmlWriter.Attributes[HtmlTextWriterAttribute.Onclick], Is.EqualTo(expectedOnClick), "Wrong OnClick");
-
-      Assert.That(_testHelper.HtmlWriter.Attributes[HtmlTextWriterAttribute.Title], Is.Not.Null, "Missing Title");
       Assert.That(_testHelper.HtmlWriter.Attributes[HtmlTextWriterAttribute.Title], Is.EqualTo(_testHelper.ToolTip), "Wrong Title");
+      Assert.That(_testHelper.HtmlWriter.Attributes[HtmlTextWriterAttribute2.Role], Is.EqualTo("button"), "Has Role");
```

Comment:
```suggestion
      Assert.That(_testHelper.HtmlWriter.Attributes[HtmlTextWriterAttribute2.Role], Is.EqualTo("button"), "Missing Role");
```

The string is the message to be displayed on failure, so we need the expectation failure.